src/test/java/org/fundacionjala/pivotal/steps/api/APIProjectSteps.java

```diff
@@ -12,17 +15,26 @@
  * Class with steps for test feature.
  */
 public class APIProjectSteps {
+    private final APIResponseName date;
+
+
+    public APIProjectSteps(final APIResponseName date) {
+        this.date = date;
+    }
+
 
     /**
      * And step to post a project.
-     *
+     * @param feature the feature.
      * @param endpoint to send request.
      * @param values the map with values to create the project.
      */
-    @And("^I post a new project to \"([^\"]*)\" endpoint$")
-    public void iPostANewProjectToEndpoint(final String endpoint, final Map<String, String> values) {
-        APICommons.setEndPoint(endpoint);
-        APIRequestManager.post(values, endpoint);
+    @And("^I post a new \"([^\"]*)\" to \"([^\"]*)\" endpoint$")
+    public void iPostANewToEndpoint(String feature, String endpoint, final Map<String, String> values){
+        //APICommons.setEndPoint(endpoint);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)